fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/RollCall.java

```diff
@@ -83,12 +85,8 @@ public void setEnd(long end) {
     this.end = end;
   }
 
-  public EventState getState() {
-    return state;
-  }
-
   public void setState(EventState state) {
-    this.state = state;
+    this.state.postValue(state);
```

Comment:
Since it is now purely event based, the UI needs only responding to a change of state due to some event (in our case a message was received that triggers an update of the state). So we put LiveData to be able to listen to the state and react to when it changes

Reply:
all right, looks good !